adapter/internal/oasparser/model/http_route.go

```diff
@@ -80,6 +83,9 @@ func (swagger *AdapterInternalAPI) SetInfoHTTPRouteCR(httpRoute *gwapiv1b1.HTTPR
 		var resourceRatelimitPolicy *dpv1alpha1.RateLimitPolicy
 		hasPolicies := false
 		var scopes []string
+		var timeoutInMillis uint32 = config.Envoy.Upstream.Timeouts.RouteTimeoutInSeconds * 1000
```

Comment:
shall we remove this from the conf as route timeouts are coming from backend CR

Reply:
@AmaliMatharaarachchi What if the `Backend` doesn't have the `timeout` property defined? If it's nil, then we need to have some default values to be set. Or else we have to omit configuring the `EndpointCluster` with the idle timeout and timeout properties which won't add the envoy headers to the enforcer response. https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/router_filter#x-envoy-retry-on

Reply:
they still would have the default values set by envoy internally, right?

Reply:
Yes, they would consider their default values. I will then conditionally configure the endpoint cluster config for the timeouts